src/platform/silabs/multi-ota/OTAMultiImageProcessorImpl.cpp

```diff
@@ -22,21 +22,22 @@
 #include <platform/DiagnosticDataProvider.h>
 #include <platform/internal/CHIPDeviceLayerInternal.h>
 #include <platform/internal/GenericConfigurationManagerImpl.h>
-
 #include <platform/silabs/multi-ota/OTAMultiImageProcessorImpl.h>
 
 using namespace chip::DeviceLayer;
 using namespace ::chip::DeviceLayer::Internal;
 
 static chip::OTAMultiImageProcessorImpl gImageProcessor;
 
-#if SL_WIFI
-#include <platform/silabs/wifi/ncp/spi_multiplex.h>
-#endif // SL_WIFI
-
 extern "C" {
+#if SL_BTLCTRL_MUX
```

Comment:
this is not useful without including the `spi_multiplex.h`

Reply:
Yes removed , will be added when multiplexing changes will be done